qiling/debugger/gdb/gdb.py

```diff
@@ -644,14 +644,21 @@ def handle_v(subcmd: str) -> Reply:
                     groups = subcmd.split(';')[1:]
 
                     for grp in groups:
-                        cmd, tid = grp.split(':', maxsplit=1)
+                        separator = ":"
+                        if (grp.find(separator) != -1):
+                            cmd, tid = grp.split(':', maxsplit=1)
+                        else:
+                            cmd = grp
 
                         if cmd in ('c', f'C{SIGTRAP:02x}'):
                             return handle_c('')
 
                         elif cmd in ('s', f'S{SIGTRAP:02x}'):
                             return handle_s('')
 
+                        else:
```

Comment:
That serves no visible purpose.

Reply:
I apologize, I'm not sure how this bit snuck past me. This was originally meant for printf debugging. I will remove this.